frontend/src/metabase/components/AddToDashSelectDashModal.jsx

```diff
@@ -24,13 +25,13 @@ export default class AddToDashSelectDashModal extends Component {
 
     static propTypes = {
         card: PropTypes.object.isRequired,
-        dashboardApi: PropTypes.object.isRequired,
         closeFn: PropTypes.func.isRequired,
         onChangeLocation: PropTypes.func.isRequired
     };
 
     async loadDashboardList() {
-        var dashboards = await this.props.dashboardApi.list({ f: "all" });
```

Comment:
👍  it feels way more straightforward to just import things like services rather than inject them into components. Feel like we should either Do-DI  or NotDo-DI, but mixing them seems like the worst of all worlds